R/formatter.r

```diff
@@ -232,32 +232,70 @@ scientific <- function(x, digits = 3, ...) {
 
 #' Ordinal formatter: add ordinal suffixes (-st, -nd, -rd, -th) to numbers.
 #'
-#' @return a function with single parameter x, a numeric vector, that
+#' @return A function with single parameter `x`, a numeric vector, that
 #'   returns a character vector
-#' @param x a numeric vector to format
+#' @param x A numeric vector of positive values to format.
+#' @param prefix,suffix Symbols to display before and after value.
+#' @param big.mark Character used between every 3 digits to separate thousands.
+#' @param rules Custom rules for computing ordinal indicators (list of regular
+#'   expressions, see examples).
+#' @param ... Other arguments passed on to [base::format()].
+#' @note
+#' Values in `x` will be rounded before formating.
 #' @export
 #' @examples
 #' ordinal_format()(1:10)
 #' ordinal(1:10)
-ordinal_format <- function() {
-  function(x) ordinal(x)
+#'
+#'  # Custom rules for French
+#' french <- list(
+#'   er = "^1$",
```

Comment:
Does this work for 101? In English it would be 101st, but in French 101e?

Reply:
Yes, in French it is "101e" and "1001e", "er" is used only for 1.